src/app/app-routing.module.ts

```diff
@@ -1,8 +1,12 @@
 import { NgModule }             from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { HomeComponent } from './home/home.component';
 
 export const routes: Routes = [
-  { path: '', redirectTo: '/indore', pathMatch: 'full'}
+    { path: '', redirectTo: 'home/indore', pathMatch: 'full'},
+    {path: 'home/:city' , component: HomeComponent },
```

Comment:
@Gurpreet-Chhabra 
Is this need to change `path: 'home/:cityName'`, As we have changed city params name?